client/session.go

```diff
@@ -33,6 +33,9 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/uber-go/tally"
+	"github.com/uber/tchannel-go/thrift"
```

Comment:
I've been relying on `goimports -local` ~ [1] for formatting which prefers 3rd party before org local. Should I revert? 

[1]: https://github.com/golang/go/issues/12420